Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -276,6 +276,21 @@ DECLARE_SOA_TABLE(UnassignedTracks, "AOD", "UNASSIGNEDTRACK",
 
 using UnassignedTrack = UnassignedTracks::iterator;
 
+// HMPID information
+namespace hmpid
+{
+DECLARE_SOA_INDEX_COLUMN(Track, track);
+DECLARE_SOA_COLUMN(HMPIDSignal, hmpidSignal, float);
+DECLARE_SOA_COLUMN(HMPIDDistance, hmpidDistance, float);
+DECLARE_SOA_COLUMN(HMPIDQMip, hmpidQMip, short);
+} // namespace hmpid
+
+DECLARE_SOA_TABLE(HMPIDs, "AOD", "HMPID",
+                  hmpid::TrackId,
+                  hmpid::HMPIDSignal,
+                  hmpid::HMPIDDistance,
+                  hmpid::HMPIDQMip);
+
```

Comment:
Last thing
Add `using HMPID = HMPIDs::iterator;`

Reply:
Hi, sure, I'm also investigating why I cannot get it to compile in the CI